### PR TITLE
[doc] fpga reproduce fail

### PR DESCRIPTION
**Pull request by h-filali**
This commit adds '--' to two of the commands to make them execute as intended.

### CODE REVIEW

File: doc/getting_started/setup_fpga.md [hunk 1]
```diff
@@ -531,8 +531,8 @@ The FPGA tests attempt to load the latest bitstream by default, but because we w
 
 ```console
 # Load the bitstream with opentitantool
-./bazelisk.sh run //sw/host/opentitantool --interface=cw340 fpga load-bitstream <path_to_your_bitstream>
+./bazelisk.sh run //sw/host/opentitantool -- --interface=cw340 fpga load-bitstream <path_to_your_bitstream>
 
 # Run the broken test locally, showing all test output and skipping the bitstream loading
-./bazelisk.sh test <broken_test_rule> --define bitstream=skip --test_output=streamed
+./bazelisk.sh test <broken_test_rule> -- --define bitstream=skip --test_output=streamed
```
Comment (pamaury) [inline doc/getting_started/setup_fpga.md:537]: I think this one is incorrect: `--define` and `--test_output` are bazel arguments so there is no need for `--`.

**Review by jwnrt — APPROVED** [review]

**Comment by jwnrt** [comment]
@engdoreis do we want to change these docs to use `--interface=hyper340` or leave them on `cw340`?

**Comment by engdoreis** [comment]
> @engdoreis do we want to change these docs to use `--interface=hyper340` or leave them on `cw340`?

I think that this doc should have the cw340 for now, the hyper340 would require extra steps which are not documented yet.

**Review by engdoreis — APPROVED** [review]
Thanks for the fix.

**Comment by jwnrt** [comment]
CI failures unrelated.

**Pull request merged**